### ISSUE TITLE
upload image with message reply

### ISSUE THREAD

**Issue by ashcrok** [LPgenerator/mattermost_bot#53]
Is there any way to attach an image (or a file in general) when replying to a message?

**Comment by SharathChandraSreenidhi** [comment]
In the plugins message.reply add image URL



@respond_to('hello$', re.IGNORECASE)
def hello_reply(message):
message.reply('*Image url* hello sender!')

**Comment by ashcrok** [comment]
Yes, that works fine. But I don't want to send the image as an URL, but as an attachment: file comes from local machine.
